app/views/_includes/applications/course-details.njk

```diff
@@ -1,4 +1,4 @@
-  <h2 class="govuk-heading-m govuk-!-margin-top-8 govuk-!-margin-bottom-2 govuk-!-font-size-27">Course applied for</h2>
+  <h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-7 govuk-!-margin-bottom-2 ">Course applied for</h2>
```

Comment:
We could make future iterations easier if we specify a heading class rather than an amalgamation of overrides